deployments/sgx_plugin/base/intel-sgx-plugin.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       - name: intel-sgx-plugin
         image: intel/intel-sgx-plugin:devel
         securityContext:
+          seLinuxOptions:
+            type: "intelplugins_t"
```

Comment:
can we pick a label so that we don't need to change it once OCP supports proper labeling?

Reply:
i don't think different label will be coming from ocp. it will be what we design and upstream (if they accept patch). naming suggestions are welcome.

Reply:
> For the plugin to run [#762 (comment)](https://github.com/intel/intel-device-plugins-for-kubernetes/issues/762#issuecomment-1075640006) was suggested so I wonder if this change is needed at all

you can ignore this comment, it wont help.

Reply:
i will update this PR after we finalize the solution in https://github.com/containers/container-selinux/pull/178

Reply:
updated to the label from container-selinux project